charger/heating/readings.go

```diff
@@ -0,0 +1,48 @@
+package heating
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evcc-io/evcc/plugin"
+)
+
+type Readings struct {
+	Power     *plugin.Config // optional
+	Energy    *plugin.Config // optional
+	Temp      *plugin.Config // optional
+	LimitTemp *plugin.Config // optional
+}
+
+func (cc *Readings) Configure(ctx context.Context) (
+	func() (float64, error),
+	func() (float64, error),
+	func() (float64, error),
+	func() (int64, error),
+	error,
+) {
+	// decorate power
+	powerG, err := cc.Power.FloatGetter(ctx)
+	if err != nil {
+		return nil, nil, nil, nil, fmt.Errorf("power: %w", err)
+	}
+
+	// decorate energy
+	energyG, err := cc.Energy.FloatGetter(ctx)
+	if err != nil {
+		return nil, nil, nil, nil, fmt.Errorf("energy: %w", err)
+	}
+
+	// decorate temp
+	tempG, err := cc.Temp.FloatGetter(ctx)
+	if err != nil {
+		return nil, nil, nil, nil, fmt.Errorf("temp: %w", err)
+	}
+
+	limitTempG, err := cc.LimitTemp.IntGetter(ctx)
+	if err != nil {
+		return nil, nil, nil, nil, fmt.Errorf("limit temp: %w", err)
+	}
```

Comment:
Was genau würde man hier für _**eine**_ Temperatur zurückgeben? Braucht es das?
Hydraulikschemata für Heizungssyteme gibt es in allen Ausführungen. Jedes ist anders. Temperaturmesspunkte gibt es daher quasi auch beliebig viele und in beliebiger Relevanz für die individuelle Anlage.
Welchen _**einen**_ würde man davon zurückgeben - und wozu?
In der Praxis müsste man dazu eine Auswahl aller im jeweiligen System möglichen Quellen hinterlegen und individuell konfigurieren. Aber wozu überhaupt? Was hilft einem z. B. die Information dass der VL von Heizkreis 2 gemäß Heizkurve aktuell 31°C hat?
Diese Daten zu verarbeiten und in entsprechende Steuerbefehle nach Anwendervorgabe umzusetzen ist ja schon Basisaufgabe einer jeden Heizungssteuerung.

Reply:
Wie gesagt- das _ist_ heute beim SG Ready charger alles schon so. Nix Neues.